custom_components/plugwise/switch.py

```diff
@@ -10,11 +10,20 @@
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
 from plugwise.nodes import PlugwiseNode
 
-from .const import CB_NEW_NODE, COORDINATOR, DOMAIN, LOGGER, PW_TYPE, SMILE, STICK, USB
+from .const import (  # pw-beta usb
```

Comment:
Not all imported constants are used by USB only.

Reply:
Whole lot of love goes to isort :(